### DISCUSSION TITLE
Change Middlewares to Middleware

### DISCUSSION THREAD

**Discussion by Temepest74** [spatie/laravel-permission#1800] [Ideas]
I think this ` \Spatie\Permission\Middlewares\RoleMiddleware ` should be changed to this ` \Spatie\Permission\Middleware\RoleMiddleware `. Laravel uses at his core "Middleware"

**Comment by erikn69** [comment]
That is an unnecessary breaking change, we would have to change our `$routeMiddleware` and `$middlewarePriority`
Maybe for the V5 version with upgrade guide
